src/components/Books.js

```diff
@@ -0,0 +1,17 @@
+import Book from './Book';
+import Form from './form';
+import Navbar from './Navbar';
+
+function Books() {
+  const title = 'Smarter way to learn React';
+  const author = 'Chun-li';
+  return (
+    <div>
+      <Navbar />
+      <Book title={title} author={author} />
```

Comment:
- [ ] This component should render a list of books using the `Book` component, kindly implement this, one way to do this is by creating an array with books and using Array.map to render all the books in the list